client/my-sites/jetpack-search/main.tsx

```diff
@@ -71,15 +77,25 @@ export default function JetpackSearchMain(): ReactElement {
 			/>
 
 			<PromoCard
-				title={ translate( 'Jetpack Search is active on your site.' ) }
+				title={
+					isSearchEnabled
+						? translate( 'Jetpack Search is active on your site.' )
+						: translate( 'Jetpack Search is disabled on your site.' )
+				}
 				image={ { path: JetpackSearchSVG } }
 				isPrimary
 			>
-				<p>{ translate( 'Your visitors are getting our fastest search experience.' ) }</p>
+				<p>
+					{ isSearchEnabled
+						? translate( 'Your visitors are getting our fastest search experience.' )
+						: translate(
+								'Enable it to ensure your visitors receive our fastest search experience.'
+						  ) }
+				</p>
 
 				<PromoCardCTA
 					cta={ {
-						text: translate( 'Settings' ),
+						text: isSearchEnabled ? translate( 'Settings' ) : translate( 'Enable now' ),
```

Comment:
I feel like if it's 'Enable now', then if user clicks this button, then the search would be enabled, which is actually not the case. But I don't have a better option. Do you have one?
Or possibly we could keep just 'Settings' button here?  :-)

Reply:
Ah, good point! Maybe `Configure now`? Otherwise I'm happy to keep it as `Settings`.

Reply:
Thanks for your prompt reply 👍 
I'm happy to keep it as `Settings` :-)

Reply:
Sure thing - fixed!